### ISSUE TITLE
managedapp create param json file error

### ISSUE THREAD

**Issue by aplihs** [Azure/azure-cli#7238]
**Describe the bug**
managedapp create errors out when using an external params.json. Got the error ...azure-cli-resource\azure\cli\command_modules\resource\custom.py", line 561, in create_application
TypeError: __init__() takes 1 positional argument but 5 were given

**To Reproduce**
`az managedapp create` with all the necessary inputs and path to params file. 

**Expected behavior**
Should work the way it works for inline params.

**Environment summary**
Tried this on cli 2.0.39 on Win 10.

**Comment by tjprescott** [comment]
@aplihs please try updating to the latest CLI as you are using an older version.

@adewaleo can you quickly try and repro this?  If there's a bug it's likely an easy fix. It reads like something is using positional args that now needs to use keyword args.

**Comment by aplihs** [comment]
still the same. This is the cmd i am running with properly structured params file
`az managedapp create --resource-group rgName --name "appName" --location "eastus2" --kind "MarketPlace" --managed-rg-id $managedGroupID --parameters "params.json" --plan-nam
e "planName" --plan-product "sku" --plan-publisher "publisherName" --plan-version "version" --debug`

and here is the params.json 
```
 {
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
     "param1": { "value": "value1" }
   }
 }
```

**Comment by adewaleo** [comment]
Hey @tjprescott @aplihs  I will look into this.

**Comment by aplihs** [comment]
@adewaleo Inline params gives the same error

**Comment by adewaleo** [comment]
Fixed in #7268
